src/Api/Auth/Controllers/AccountsController.cs

```diff
@@ -471,7 +471,7 @@ public async Task<ProfileResponseModel> PutProfile([FromBody] UpdateProfileReque
 
         var twoFactorEnabled = await _userService.TwoFactorIsEnabledAsync(user);
         var hasPremiumFromOrg = await _userService.HasPremiumFromOrganization(user);
-        var organizationIdsManagingActiveUser = await GetOrganizationIdsManagingUserAsync(user.Id);
+        var organizationIdsManagingActiveUser = await GetOrganizationIdsClaimingUserAsync(user.Id);
```

Comment:
Does this var `organizationIdsManagingActiveUser` need to be renamed as well?

Reply:
local variables strike again. I really thought I had got all the `Manag*` spots :(